modules/contact/test/tests/bouncing-block-contact/grid-sequencing/grid-sequencing.i

```diff
@@ -0,0 +1,224 @@
+starting_point = 2e-1
+
+# We offset slightly so we avoid the case where the bottom of the slave block and the top of the
+# master block are perfectly vertically aligned which can cause the backtracking line search some
+# issues for a coarsest mesh (basic line search handles that fine)
+offset = 1e-2
+
+[GlobalParams]
+  displacements = 'disp_x disp_y'
+  diffusivity = 1e0
+  scaling = 1e0
+[]
+
+[Mesh]
+  file = level0.e
+[]
+
```

Comment:
So right now we suppose that users will provide the coarsest mesh? Right?

It may be helpful to add a bit doc on this.